src/commands/hardis/doc/object-field-usage.ts

```diff
@@ -0,0 +1,393 @@
+import { Flags, SfCommand, requiredOrgFlagWithDeprecations } from '@salesforce/sf-plugins-core';
+import { AnyJson } from '@salesforce/ts-types';
+import { Connection } from '@salesforce/core';
+import c from 'chalk';
+import sortArray from 'sort-array';
+import { generateReports, uxLog, uxLogTable } from '../../../common/utils/index.js';
+import { soqlQuery, soqlQueryTooling } from '../../../common/utils/apiUtils.js';
+
+type FieldUsageRow = {
+  sObjectName: string;
+  fieldApiName: string;
+  fieldLabel: string;
+  totalRecords: number;
+  populatedRecords: number;
+  populatedPercentageNumeric: number;
+  populatedPercentage: string;
+};
+
+type SkippedFieldInfo = {
+  fieldName: string;
+  reason: string;
+};
+
+export default class HardisDocObjectFieldUsage extends SfCommand<any> {
+  public static description = `
+## Command Behavior
+
+**Analyzes how populated fields are for a specific Salesforce object.**
+
+This command focuses on a single sObject and measures how many records populate each non-required field. It is useful for understanding data completeness before refactoring, cleaning up unused fields, or preparing migration plans.
+
+- **Target Org:** Use \`--target-org\` to pick the org connection context.
+- **Single sObject:** Provide the API name of the object to analyze via \`--sObject\`.
+- **Per-field Counts:** Performs one overall record count and one per-field count with \`SELECT COUNT() FROM <sObject> WHERE <field> != null\`, skipping required or non-filterable fields.
+- **Single-field Distribution:** Combine \`--sObject\` with \`--field\` to group by that field and list distinct values with their record counts and usage percentages.
+- **Reporting:** Generates CSV/XLSX reports and prints a summary table with per-field population rates.
+`;
+
+  public static examples = [
+    '$ sf hardis:doc:object-field-usage --sObject Account',
+    '$ sf hardis:doc:object-field-usage --target-org myOrgAlias --sObject CustomObject__c',
+    '$ sf hardis:doc:object-field-usage --sObject Account --field SalesRegionAcct__c',
+  ];
+
+  public static flags: any = {
+    'target-org': requiredOrgFlagWithDeprecations,
+    sObject: Flags.string({
```

Comment:
plz can u use 's-objects' or 'objects' ? :)
It would be nice to be able to call the command for multiple objects in one call

In case the option is not sent, you can also prompt the user to select objets that he/she wants to analyze :)

See example of objects selection here: https://github.com/hardisgroupcom/sfdx-hardis/blob/e346bfcd7444c6730950d8af9e2313b0604e6f05/src/commands/hardis/org/diagnose/storage-stats.ts#L202